src/room.py

```diff
@@ -2,30 +2,37 @@
 # description attributes.
 
 class Room:
-    def __init__(self, name, description, items, baddies):
+    def __init__(self, name, description, items, baddies, weapons):
         self.name = name
         self.description = description
         self.items = items
         self.baddies = baddies
+        self.weapons = weapons
         self.n_to = None
         self.s_to = None
         self.e_to = None
         self.w_to = None
     
     def __str__(self):
-        stuff = f"\nItems here: "
+        stuff = f"\n Items here: "
         if len(self.items)>0:
             for i in self.items:
                 stuff += f" {i}"
         else:
             stuff += "None"
-        enemies = f"\nEnemies here: "
+        enemies = f"\n Enemies here: "
```

Comment:
You can make logic here more readable by declaring new Room Class functions instead.